containers/buildAlpaquita1.docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM bellsoft/alpaquita-linux-base:stream-musl
+
+MAINTAINER Max Neunhoeffer <hackers@arangodb.com>
+
+ARG ARCH="x86_64"
+
+COPY ./scripts /scripts
+
+RUN /scripts/install-alpine318-repository.sh
+
+RUN apk add groff bison flex make cmake ccache python3 git linux-headers vim boost-dev ctags mandoc gdb fish openssh db-dev file libltdl zlib-dev curl coreutils texinfo gcovr py-setuptools valgrind libtool nodejs npm py3-psutil && apk add g++ prometheus bash liburing-dev clang clang16 clang16-static clang16-dev clang16-analyzer lld
```

Comment:
Wondering if the `lld` in this list should rather be `lld-16`, simply because the other clang packages are all versioned.

Reply:
Good point, but it must be lld, strangely enough.